src/yaml_configs/customization_compiled_in.rs

```diff
@@ -212,6 +212,31 @@ code_lens:
         any tricky parts in the code. Be concise, wait for a more specific follow-up question from the user.
 
 
+custom_cmdline_tools:
+  cargo_check:
+    description: "Checks for cargo errors"
+    parameters:
+      - name: "path"
+        type: "string"
+        description: "absolute path to the project"
+    parameters_required:
+      - "path"
+    command: "cargo check --manifest-path %path%/Cargo.toml"
+    runs_in_background: false
+    runs_in_background_false_timeout: 120
+    experimental: true
+
+  run_chat_js:
+    description: "runs chat js in background"
+    parameters:
+      - name: "path"
+        type: "string"
+        description: "absolute path to the project"
+    parameters_required:
+    command: "npm --prefix %path% run dev"
+    runs_in_background: true
+    experimental: true
+
```

Comment:
Don't need this for everybody all the time.

But we might need it in default integrations.yaml